_posts/2016-05-25-coordination-in-akka.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+name: Coordination-in-akka
+title: 'Coordination in Akka'
+date: 2016-05-25 00:20:00 +00:00
+author: Felipe Fernández
+image:
+    src: /assets/img/custom/blog/coordination.jpg
+    attribution:
+      text: Castellers! by Nacho Pintos
+      href: https://www.flickr.com/photos/bonus1up/3727496558
+tags:
+- akka
+- coordination
+- supervision
+- akka-http
+---
+
+This is the fifth post in the series about integrating sync clients with async systems ([1, ](http://codurance.com/2016/04/28/async-systems-with-sync-clients/)[2, ](http://codurance.com/2016/04/30/akka-basics/)[3, ](http://codurance.com/2016/05/10/finite-state-machines-with-akka/)[4](http://codurance.com/2016/05/16/publish-subscribe-model-in-kafka/)). Here we'll see how to manage the actor's lifecycle so our service can use efficiently the available resources.
```

Comment:
can use efficiently the available resources. ==> can use the available resources efficiently. or can make efficient use of the available resources